Jenkinsfile

```diff
@@ -1,96 +1,21 @@
 #!/usr/bin/env groovy
 
-REPOSITORY = 'frontend'
-DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
-
 node('mongodb-2.4') {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
 
-  properties([
-    buildDiscarder(
-      logRotator(
-        numToKeepStr: '50')
-      ),
-    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-    [$class: 'ThrottleJobProperty',
-      categories: [],
-      limitOneJobWithMatchingParams: true,
-      maxConcurrentPerNode: 1,
-      maxConcurrentTotal: 0,
-      paramsToUseForLimit: 'frontend',
-      throttleEnabled: true,
-      throttleOption: 'category'],
-    [$class: 'ParametersDefinitionProperty',
-      parameterDefinitions: [
-        [$class: 'BooleanParameterDefinition',
-          name: 'IS_SCHEMA_TEST',
-          defaultValue: false,
-          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
-        [$class: 'StringParameterDefinition',
-          name: 'SCHEMA_BRANCH',
-          defaultValue: DEFAULT_SCHEMA_BRANCH,
-          description: 'The branch of govuk-content-schemas to test against']]
-    ],
-  ])
-
-  try {
-    if (!govuk.isAllowedBranchBuild(env.BRANCH_NAME)) {
-      return
-    }
-
-    stage("Checkout") {
-      checkout scm
-    }
-
-    stage("Clean up workspace") {
-      govuk.cleanupGit()
-    }
-
-    stage("git merge") {
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Configure Rails environment") {
-      govuk.setEnvar("RAILS_ENV", "test")
-    }
-
-    stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency(params.SCHEMA_BRANCH)
-      govuk.setEnvar("GOVUK_CONTENT_SCHEMAS_PATH", "tmp/govuk-content-schemas")
-    }
-
-    stage("bundle install") {
-      govuk.bundleApp()
-    }
-
-    stage("rubylinter") {
-      govuk.rubyLinter("app test lib")
-    }
-
-    stage("Precompile assets") {
-      govuk.precompileAssets()
-    }
-
-    stage("Run tests") {
-      govuk.runRakeTask("ci:setup:rspec default")
-    }
-
-    if (env.BRANCH_NAME == "master") {
-      stage("Push release tag") {
-        govuk.pushTag(REPOSITORY, env.BRANCH_NAME, "release_${env.BUILD_NUMBER}")
+  govuk.buildProject(
+    sassLint: false,
+    beforeTest: {
+      // TODO: Make this the default in Jenkinslib and override in individual
+      // projects that need `env.RACK_ENV`
+      stage("Test setup") {
+        govuk.setEnvar("RACK_ENV", "")
       }
-
-      stage("Deploy to integration") {
-        govuk.deployIntegration(REPOSITORY, env.BRANCH_NAME, 'release', 'deploy')
+    },
+    testTask: {
+      stage("Run tests") {
+        govuk.runRakeTask("ci:setup:rspec default")
```

Comment:
This actually looks like it could run the normal `default` rake task. I think I confused this repo with `static`, where we can't run the test tasks as part of `default`:

https://github.com/alphagov/static/blob/master/Jenkinsfile#L65-L66

Tried it in https://github.com/alphagov/static/pull/954.